k8s_local/adservice.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: adservice
+  labels:
+    app: adservice
+    component: otel-demo-adservice
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: adservice
+      component: otel-demo-adservice
+  template:
+    metadata:
+      labels:
+        service: adservice
+        app: adservice
+        component: otel-demo-adservice
+    spec:
+      containers:
+        - name: adservice
+          image: otel/demo:${DOCKER_IMAGE_VERSION}-adservice
```

Comment:
following https://github.com/open-telemetry/opentelemetry-demo-webstore/issues/156, the package is now public, we could use `ghcr.io/open-telemetry/demo` now.